connectors/overview.mdx

```diff
@@ -112,6 +112,32 @@ fn my_to_json(f: f64) -> String {
 used for storing data in data lakes. Arroyo supports writing Parquet via the FileSystem
 sink. Refer for the [FileSystem sink docs](/connectors/filesystem) for details.
 
+## Framing
+
+The framing configuration for a source determines how messages read from the source
+are split into records for processing. By default, this is set to `none`, which
+means that the source will emit a single record for each message it reads. This is
+particularly useful for sources—like HTTP endpoints—that do not have their own
```

Comment:
This is confusingly worded. Are you saying that the default "none" is "particularly useful"? Given that the base assumption would be this behavior, feels weird to describe it as "particularly useful". Is this supposed to describe situations where you'd want framing?

Reply:
You're right, I think it's clearer if the order of sentences is swapped ("This" is meant to refer to the concept of framing, not to `none`).